src/auth.ts

```diff
@@ -11,6 +11,13 @@ const env0SecretKey = "env0.secret";
 
 export class AuthService {
   constructor(private readonly context: vscode.ExtensionContext) {}
+
+  private selectedOrgId: string | undefined;
+
+  public getSelectedOrg() {
+    return this.selectedOrgId;
+  }
```

Comment:
I totally agree.
I did that in this way, because we do the same for user credentials.
When I created auth service i wanted to pass the instance of auth service to the api client to avoid this problem but i got this [**comment**](https://github.com/env0/vscode-env0/pull/37#discussion_r1265406717) 
4c8a4963f8d07974919d697e7ea64ed6af65070e